config.go

```diff
@@ -21,6 +21,8 @@ type Config struct {
 	RootURL string
 	// BCryptCost is the cost of the bcrypt password hashing function.
 	BCryptCost int
+	// If true, authboss won't use any goroutines. Dependencies of authboss may or may not use goroutines.
+	DisableGoroutines bool
```

Comment:
I think the comment should read:
"DisableGoroutines stops authboss from spawning any goroutines inside the library for example to send mail".

Also, I'd like to push back on this flag, it's an anti-feature. Enabling it makes your app strictly worse as it doesn't follow best practices of isolating your background tasks from your request response. Is GAE really this limiting? Do we have no other workaround?